### PR TITLE
Expose input changes event

### LINKED ISSUE #30: Event when input changes the value
Hi,

I used your component and it's great. But then i ran into a problem that i need to listen to the input changes event, for example, if the input value is empty, my "Action" button will be disabled.

Unfortunately, there's no exposed function for that event yet.

### PR DESCRIPTION
**Pull request by khanghoang**
Close #30

### CODE REVIEW

File: example/src/app.js [hunk 1]
```diff
@@ -42,6 +43,10 @@ var App = React.createClass({ // eslint-disable-line
     console.log('onBlur'); // eslint-disable-line
   },
 
+  onInputChange: function(value) {
```
Comment (ro-ka) [inline example/src/app.js]: Better name it `onChange`, to be in line with `onFocus` and `onBlur`.

File: src/Geosuggest.jsx [hunk 2]
```diff
@@ -24,7 +24,8 @@ var Geosuggest = React.createClass({
       types: null,
       googleMaps: google && google.maps,
       onFocus: noop,
-      onBlur: noop
+      onBlur: noop,
+      onInputPress: noop,
```
Comment (ro-ka) [inline src/Geosuggest.jsx]: Name it consistently `onChange`.

File: src/Geosuggest.jsx [hunk 3]
```diff
@@ -61,6 +63,7 @@ var Geosuggest = React.createClass({
    */
   update: function (value) {
     this.setState({userInput: value});
+    this.props.onInputChange(userInput)
```
Comment (ro-ka) [inline src/Geosuggest.jsx]: The param is called `value` here.

**Comment by ro-ka** [comment]
Please run `npm test` before pushing and adjust the ES Lint errors:

```
./src/Geosuggest.jsx
28:24  error    Trailing comma                               no-comma-dangle
28:24  error    Unexpected trailing comma                    comma-dangle
56:41  error    Missing semicolon                            semi
66:29  error    "userInput" used outside of binding context  block-scoped-var
66:29  error    "userInput" is not defined                   no-undef
66:39  error    Missing semicolon                            semi
```

Will merge when it passes the test and after renaming the function name.
Thanks! :)

**Comment by khanghoang** [comment]
@ro-ka done :)

**Comment by ro-ka** [comment]
Great, thank you!

**Comment by ro-ka** [comment]
Released in `1.9.0`.

**Pull request merged**